core/jazz_cloud-logs-streamer/index.js

```diff
@@ -41,8 +41,9 @@ module.exports.handler = (input, context, cb) => {
 
     // decompress the input
     zlib.gunzip(zippedInput, function (error, buffer) {
-        if (error) { context.fail(error); return; }
-
+        if (error) { context.fail(error); 
+            return callback(JSON.stringify(errorHandler.throwNotFoundError("100", "Context Failed Error")));
```

Comment:
We are still treating this as a microservice. Don't throw "http" errors as there is no client.